scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SchedulerController.java

```diff
@@ -223,6 +235,62 @@ public SchedulerModel getModel() {
         return this.model;
     }
 
+    public void resetUpdateStatus() {
+        this.statusUpdated = false;
+        this.executionsUpdated = false;
+        this.usersUpdated = false;
+        this.statsUpdated = false;
+        this.myAccountUpdated = false;
+        this.executionsDataUpdated = false;
+    }
+
+    public boolean allUpdated() {
+        return statsUpdated && executionsUpdated && executionsDataUpdated && usersUpdated && statsUpdated &&
+               myAccountUpdated;
+    }
+
+    public void setStatusUpdated(boolean statusUpdated) {
+        this.statusUpdated = statusUpdated;
+        if (allUpdated()) {
+            restartTimer();
+        }
+    }
+
+    public void setExecutionsUpdated(boolean executionsUpdated) {
+        this.executionsUpdated = executionsUpdated;
+        if (allUpdated()) {
+            restartTimer();
+        }
+    }
+
+    public void setExecutionsDataUpdated(boolean executionsDataUpdated) {
+        this.executionsDataUpdated = executionsDataUpdated;
+        if (allUpdated()) {
+            restartTimer();
+        }
+    }
+
+    public void setUsersUpdated(boolean usersUpdated) {
```

Comment:
why public not private?

Reply:
it can be private, the same for setStatusUpdated, setStatsUpdated. I will change them

Reply:
some of them had to be public as they are accessed from outside this class

Reply:
yes, I have just changed the ones that are accessed  from inside the class

Reply:
OK LGTM otherwise. I didn't verify though if there are missing spots where we should set<VAR>Updated(true). I approve